deploy/config.yaml

```
@@ -0,0 +1,8 @@
+deploymentMode: "SelectorSyncSet" 
+selectorSyncSet: 
+  resourceApplyMode: "Sync"
```

```suggestion
  resourceApplyMode: "Upsert"
```
Should this be `Upsert` instead? That's what was there previously: https://github.com/openshift/managed-cluster-config/pull/2157/files#diff-d80f8bf0f9ac8d5f85a7a45a38c33f50f9925fd44e75043151bd6820d7656d80L3, though I'm not sure if anything else modifies this namespace's labels

---

Ah ok, it's so we don't accidentally delete the namespace when we delete this syncset https://github.com/openshift/hive/blob/master/docs/syncset.md - imo yes we should change this to `Upsert`

/hold

---

This was changed to Upsert during the migration and was originally `Sync`

/unhold
/approve